OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -3034,4 +3155,124 @@ static void fireEmailUpdateFailure() {
          emailUpdateHandler = null;
       }
    }
+
+   /*
+   * Start OneSignalOutcome module
+   */
+
+   private static OutcomeSettings outcomeSettings = null;
+
+   public static void changeOutcomeSettings(OutcomeSettings settings){
```

Comment:
Does look like this is used anywhere. Also if we do need it it should not be public

Reply:
It looks like this was used to mock out the settings for tests in the past but they were changed to not use it any more. Android Studio now says it isn't called any where any more

Reply:
`cacheActive` should always be enabled so outcomes can always be set offline. Don't see a case where it would be different from app to app.

Reply:
We can leave it as package-private for now so the code is there, we can open the API up later if it is really needed.